guides/source/getting_started.md

```diff
@@ -186,6 +186,7 @@ of the files and folders that Rails creates by default:
 |test/|Unit tests, fixtures, and other test apparatus. These are covered in [Testing Rails Applications](testing.html).|
 |tmp/|Temporary files (like cache and pid files).|
 |vendor/|A place for all third-party code. In a typical Rails application this includes vendored gems.|
+|.gitattributes|This file defines certain attributes to pathnames in a git repository. See [Git - gitattributes](https://git-scm.com/docs/gitattributes) for more info about git attribute files.|
```

Comment:
([This is the file we generate](https://github.com/rails/rails/blob/ce1806d945755f0c68e5bbc66d85e53d8c05d371/railties/lib/rails/generators/rails/app/templates/gitattributes.tt), for anyone curious.)

I feel like, if someone did not already know about `.gitattributes`, they would still not understand its purpose after reading this description. (Compare to the description for `.gitignore`.)  What do you think about the following?

```suggestion
|.gitattributes|This file defines metadata for specific paths in a git repository. This metadata can be used by git and other tools to enhance their behavior. See the [gitattributes documentation](https://git-scm.com/docs/gitattributes) for more information.|
```

Reply:
Yes, this is more descriptive. I'll make this change 👍

Reply:
I've also changed `info` to `information` in the description for `.gitignore`.